frontier/core/src/main/java/com/zhaw/frontier/screens/PauseScreen.java

```diff
@@ -0,0 +1,125 @@
+package com.zhaw.frontier.screens;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.InputMultiplexer;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.badlogic.gdx.utils.viewport.ScreenViewport;
+import com.zhaw.frontier.FrontierGame;
+
+/**
+ * Shows a pause screen with options to resume, save, or save and exit the game.
+ */
+public class PauseScreen implements Screen {
+
+    private FrontierGame frontierGame;
+    private Stage stage;
+    private Skin skin;
+
+    public PauseScreen(FrontierGame frontierGame) {
```

Comment:
```suggestion
    public PauseScreen(FrontierGame frontierGame, GameScreen gameScreen) {
       this.gameScreen = gameScreen;
```


we also need to save the game screen, since we want to open it with the same values again.

Reply:
@TheBromo I am wondering isn't it a good idea to store and load the GameScreen values from the FrontierGame? I am thinking of something similar to PM3 where we where storing all the Information for the game in the gamestate.

Reply:
yes this can also be done.